### ISSUE TITLE
SimpleJekyllSearch / jekyll-search.js is throwing errors (v4)

### ISSUE THREAD

**Issue by petetnt** [twbs/bootstrap#17200]
Bootstrap (/), Documentation and Examples-pages are throwing this:

```
Uncaught ReferenceError: se is not defined
   (anonymous function) @ jekyll-search.js:1
```

and as (/) and Examples don't contain a search input element, they are also throwing a searchInputthrowError :

```
jekyll-search.js:1 Uncaught Error: SimpleJekyllSearch --- You must specify a searchInputthrowError 
  @ jekyll-search.js:1validateOptions
  @ jekyll-search.js:1self.init
  @ jekyll-search.js:1(anonymous function) @ (index):292
```

(Searching does seem to work properly on the docs page however)

**Comment by XhmikosR** [comment]
I guess I'll try including the search JS only where it's used. PR pending.

**Comment by XhmikosR** [comment]
All right, I fixed it in my branch, but I can't split it right now.

https://github.com/twbs/bootstrap/commit/7d0128e7d6a163f8014da623c6cd88aa9ee78d3c

**Comment by petetnt** [comment]
@XhmikosR awesome :+1:

**Comment by XhmikosR** [comment]
Just for reference, the first error is caused by wrong minification due to th `-se` part in the JS file.
The second was just because the JS was included everywhere but not the needed initialization code. So now I include it where we include the library itself.

**Comment by XhmikosR** [comment]
Merged #17384.
